appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -44,6 +44,8 @@ Blockly.Msg.en.switch_language_to_english = {
     Blockly.Msg.UNDO = 'Undo';
     Blockly.Msg.REDO = 'Redo';
     Blockly.Msg.CLEAN_UP = 'Clean up Blocks';
+    Blockly.Msg.HIDE = 'Hide Workspace';
```

Comment:
I think we should have the text here be "Hide Workspace Controls" and "Show Workspace Controls". Hide/Show Workspace seem too generic to me.